docs/index.rst

```diff
@@ -109,15 +109,15 @@ as comments in the code generated by Xcode for the model consumer.
     coreml_model.short_description = 'Predicts the price of a house in the Seattle area.'
 
     # Set feature descriptions manually
-    model.input_description['bedroom'] = 'Number of bedrooms'
-    model.input_description['bathrooms'] = 'Number of bathrooms'
-    model.input_description['size'] = 'Size (in square feet)'
+    coreml_model.input_description['bedroom'] = 'Number of bedrooms'
```

Comment:
This change looks incorrect, since `coreml_model` has not been defined above. 
The existing code example is correct.